TicTacToe/Program.cs

```diff
@@ -9,238 +10,292 @@ namespace TicTacToe
 {
     class Program
     {
+        const int startPosition = 2;
+        const int spaceBetween = 4;
+        const int numOfBoards = 10;
+        const int innerCells = 3;
+        const int setLocation = 38;
+        const int cellsNumber = innerCells * innerCells;
+        const int distanceBoards = innerCells * spaceBetween;
+        const char player1 = 'X';
+        const char player2 = 'O';
+        const char empty = ' ';
+        static Random rand = new Random();
         static void Main(string[] args)
         {
-            char[][] board = new char[10][];
-            char player1 = 'X';
-            char player2 = 'O';
-            char currentPlayer = 'X';
-            int z = 9;
+            char[][] board = new char[numOfBoards][];
+            char currentPlayer = player2;
+            int z;
 
-            buildBoard();
-            initBoard(board);
+            HomePage();
+            InitBoard(board);
+            BuildBoard(board);
 
-            do
+            while (!SubGameOver(cellsNumber, board, currentPlayer)) 
             {
+                currentPlayer = ChangePlayer(currentPlayer);
                 if (currentPlayer == player1)
                 {
-                    z = humanTurn(board, player1);
+                    z = HumanTurn(board, player1);
                 }
                 else
                 {
-                    z = computerTurn(board, player2);
+                    z = ComputerTurn(board, player2);
                 }
-                updateBoard(board);
-                subGameOver(z, board, currentPlayer);
-                currentPlayer = changePlayer(currentPlayer);
+                SubGameOver(z, board, currentPlayer);
+                BuildBoard(board);
             }
-            while (!subGameOver(9, board, currentPlayer));
 
             Console.Clear();
-            if (currentPlayer == player1)
+            if (Draw(cellsNumber, board))
+            {
+                DrawArt();
+            }
+            else if (currentPlayer == player1)
```

Comment:
You're technically correct that if a game ended on the turn of some player, that player couldn't have lost so it's either a tie or that player won; however that's a special property of the tic-tac-toe game and in my opinion the code would be more readable if the "SubGameOver" method would return the player than won, or an indication of tie, or an indication there sub game isn't over. This case calls for using C# enums :) more self-learning?